module-2-html/exercises/02-book/sbicanic/book

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Table of contents</title>
+</head>
+<body>
+    <h1><b>1. Markup languages</b></h1>
```

Comment:
Nije potrebno koristiti `<b>` ili `<strong>` elemente na naslovima zato što su i semantički i vizualno istaknuti samim time što su naslovi.

```suggestion
    <h1>1. Markup languages</h1>
```